test/views/juttle-view.spec.js

```diff
@@ -0,0 +1,24 @@
+require('chai').should();
+var JuttleView = require('../../src/views/juttle-view');
+var v = require('../../src/lib/object-validation');
+
+describe('juttle-view', function() {
+    it('getFlattenedParamValidationErrors', function() {
+        var obj = {
+            propertyB: 1,
+            propertyC: "bob"
+        };
```

Comment:
Broke this test into a few tests including one that has an error on a nested param.